### PR TITLE
Feature/user table data

### PR DESCRIPTION
**Pull request by ccali11**
Ignore branch name. Did the following three things per the commits:
- Update Staking component with current staked by address
- Changed the positions of ETH and USD on Breakdown Overview component
- Fixed bug in calculations of Breakdown Overview component metrics

### CODE REVIEW

File: apps/web/src/pages/overview/components/BreakdownChart.vue [hunk 1]
```diff
@@ -135,11 +135,11 @@ watch(selectedTimeframe, () => {
           Total Balance Across Connected Wallets
         </h6>
         <div class="flex items-center gap-[12px]">
-          <h5 class="blance_amount">
-            {{ totalWalletBalance.usd }}
+          <h5 class="blance_eth">
```
Comment (shanejearley) [inline apps/web/src/pages/overview/components/BreakdownChart.vue:138]: "blance" must have been a typo should we find and replace?

File: apps/web/src/pages/overview/components/Staking.vue [hunk 2]
```diff
@@ -195,12 +210,13 @@ const handleDeposit = async () => {
...
 
-  }, 3000)
+  }, 2500)
 
   setTimeout(() => {
     stakingActionLoader.value = false
-  }, 3500)
+  }, 3000)
 
+  currentUserStake.value = await getUserStakeAtAddress(selectedWalletAddress.value as string)
```
Comment (shanejearley) [inline apps/web/src/pages/overview/components/Staking.vue:219]: Is `getUserStake` already taken? AtAddress seems redundant but I could be missing context.

**Review by shanejearley — APPROVED** [review]
Very nice changes.

**Comment by ccali11** [comment]
Thanks for comments. Will resolve and merge. 👍🏻

File: apps/web/src/pages/overview/components/Staking.vue (hunk 2 above)
Comment (ccali11) [reply apps/web/src/pages/overview/components/Staking.vue:219]: @shanejearley - `getUserStake` could mean "get user's stake across all addresses associated with them", so I just more narrowly defined it as get the stake at a particular address of theirs. With that context, lmk if you want me to still change it.

Comment (shanejearley) [reply apps/web/src/pages/overview/components/Staking.vue:219]: I see. No big preference. If we call the broader cases that use summing getTotalUserStake and getTotalUserBalance that could also work.

Comment (ccali11) [reply apps/web/src/pages/overview/components/Staking.vue:219]: Ok. I will do that. 👍

**Pull request merged**